core/src/main/java/org/neo4j/ogm/metadata/FieldInfo.java

```diff
@@ -475,6 +477,17 @@ public boolean isVersionField() {
         return field.getAnnotation(Version.class) != null;
     }
 
+    /**
+     * @return True if this field info describes the internal identity field.
+     */
+    public boolean isInternalIdentity() {
```

Comment:
This method can get package private visibility.

Reply:
Good point. Thanks and done.